src/core/etl/src/Flow/ETL/Row/Entry/XMLEntry.php

```diff
@@ -119,7 +118,7 @@ public function rename(string $name) : Entry
     public function toString() : string
     {
         /** @phpstan-ignore-next-line */
-        return $this->value->saveXML();
+        return $this->value->saveXML($this->value->documentElement);
```

Comment:
This removes additional `<?xml version="1.0" ?>` being added to nodes.

Reply:
Why do we want to remove it?

Reply:
As you can see in tests, by default `\DOMDocument` adds `<?xml` tags to everything, even if you extract a particular node of the XML which looks invalid IMO.